spring-6-rsocket/pom.xml

```diff
@@ -33,6 +33,26 @@
             <type>pom</type>
             <scope>import</scope>
         </dependency>
```

Comment:
@Bipinkumar27 

Try to delete these 4 dependencies and check that build/integration works.
Of course leave the properties tag changes.

Reply:
It worked !! I have done the changes